react/README.md

```diff
@@ -34,12 +34,28 @@ For example, the following works:
 <MaskedTextInput
   mask={['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]}
   className="form-control"
-  placeholder="Enter zip code"
+  placeholder="Enter a phone number"
   guide={false}
   id="my-input-id"
 />
 ```
 
+You can additionally use your own custom input component by passing in the prop `inputElement`.
```

Comment:
Hmmm, I was just referring to the wording in the `README.md`. But yeah, `createTextMaskInputElement` expects the `inputElement` to be an HTML input element, so I'm not sure how that would work...